### PR TITLE
Updated Workflow

### PR DESCRIPTION
**Pull request by mridang**
## Checklist:
- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] I have assigned the correct milestone or created one if non-existent.
- [ ] I have correctly labeled this pull request.
- [ ] I have linked the corresponding issue in this description.
- [ ] I have updated the corresponding Jira ticket.
- [ ] I have requested a review from at least 2 reviewers
- [ ] I have checked the base branch of this pull request
- [ ] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

**Comment by supercid** [comment]
We need to squash or use git --fix-up to trim the history a bit
https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -3,17 +3,154 @@ name: PHP CI
...
+        name: phpmd-xml-result
+        path: pmdphpmd.xml
 
+    - name: Report annotations
+      id: report-annotations
+      run: ./vendor/bin/pmd2pr --graceful-warnings pmdphpmd.xml
+
+  package:
+    name: Package
+    runs-on: ubuntu-latest
```
Comment (supercid) [inline .github/workflows/action.yml:114]: Just as a question, but can't we use alpine or some slim image? Might speed up a bit.

File: Console/Command/NostoAccountConnectCommand.php [hunk 2]
```diff
@@ -231,8 +231,7 @@ private function getTokensFromInput(InputInterface $input, SymfonyStyle $io)
         $emailToken = $input->getOption(Token::API_EMAIL . self::TOKEN_SUFFIX);
         if (!$emailToken && $this->isInteractive) {
             $emailToken = $io->ask(
-                'Enter Email Token (Optional): ',
-                false
```
Comment (olsi-qose) [inline Console/Command/NostoAccountConnectCommand.php:235]: why is `false` removed?

Comment (supercid) [reply Console/Command/NostoAccountConnectCommand.php:235]: Default param

Comment (olsi-qose) [reply Console/Command/NostoAccountConnectCommand.php:235]: 👍

**Pull request merged**